ci/post/main.py

```diff
@@ -62,22 +62,41 @@ def _match_version_number(text, regex):
 	return int(match.group(1))
 	
 
-def get_source_version(date_version):
+def get_source_version(date_version: datetime, tag_name: str) -> semantic_version.Version:
 	"""! Retrieves the build's version from `version.cmake`, forms it as a string, and appends the date
 
 	@param[in] `date_version` Date this build is published for release
+	@param[in] 'tag_name'     Tag of this release, used to determine if release, rc, or nightly
 
-	@return String of the form `MAJOR_VERSION.MINOR_VERSION.BUILD_VERSION-date`
+	@return String of the form `MAJOR_VERSION.MINOR_VERSION.BUILD_VERSION-date`, ' MAJOR_VERS
```

Comment:
Is `MAJOR_VERS` supposed to be `MAJOR_VERSION`?

Reply:
blah, I got interrupted and forgot that I was mid-sentence.  Its supposed to document the three variations of version string it can produce